src/main/java/com/cars/carlist/CarController.java

```diff
@@ -6,18 +6,17 @@
 import java.util.List;
 
 @RestController
-public class CarsController {
+public class CarController {
 
-    private final CarsService carsService;
+    private final CarService carService;
 
-    public CarsController(CarsService carsService) {
-        this.carsService = carsService;
+    public CarController(CarService carService) {
+        this.carService = carService;
     }
 
     @GetMapping("/cars")
-    public List<Cars> getNames() {
-        List<Cars> names = carsService.getNames();
+    public List<Car> getAll() {
```

Comment:
CarでなくCarResponseにする。
ResponseはHTTPの話なので業務ロジックを持つServiceやDBとの関連が強いMapperには依存しないように実装したい。
```suggestion
    public List<CarResponse> getAll() {
```

Reply:
こちらについてはResponseクラスが出来上がったら変更するつもりでそのままになっておりました。
ひとまず、変更いたしました。
<img width="779" alt="スクリーンショット 2023-09-15 22 55 51" src="https://github.com/sugao-2211/task9/assets/141313076/531f7a88-73cb-418a-a4ad-75e5cbae72f6">